roda-app.rb

```diff
@@ -15,7 +15,7 @@
 class RodaApp < Roda
   # bots init
   opts[:reservation_bot] = Slackbotsy::Bot.new(ReservationSlackConfig::CONFIG)
-  plugin :assets, css: 'signin.css'
+  plugin :assets, css: ['signin.css', 'reservations.css']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.